ospd_openvas/nvticache.py

```diff
@@ -89,7 +92,13 @@ def get_oids(self) -> Iterator[Tuple[str, str]]:
             A i. Each single list contains the filename
             as first element and the oid as second one.
         """
-        return OpenvasDB.get_filenames_and_oids(self.ctx)
+        if self.notus:
+            for f, oid in self.notus.get_filenames_and_oids():
+                yield (f, oid)
+        if self.ctx:
+            for f, oid in OpenvasDB.get_filenames_and_oids(self.ctx):
+                if not self.notus or not self.notus.exists(oid):
+                    yield (f, oid)
```

Comment:
What does the A i. in the docstring mean? I think the docstring is wrong as we return a Tuple, not a List

Reply:
It is actually returning a generator / iterator of a tuple so list seems kind of matching to me. But going to change it